streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -266,7 +268,7 @@ void retryWithBackoff(final Map<TaskId, Set<TopicPartition>> tasksToBeCreated, f
                     } catch (final LockException e) {
                         // ignore and retry
                         if (!retryingTasks.contains(taskId)) {
-                            log.warn("{} Could not create task {}. Will retry: ", logPrefix, taskId, e);
+                            log.warn("{} Could not create task {} due to {}. Will retry: ", logPrefix, taskId, e);
```

Comment:
I thought, one would need to do an explicit `e.toString()` to get this behavior:

> log.warn("{} Could not create task {} due to {}. Will retry: ", logPrefix, taskId, e.toString());

I think it would be good to not print the whole stack trace, as many users are super worries about this WARN level message -- IMHO, StackTrace should only be included for ERROR, but not for WARN messages.